setup.sh

```diff
@@ -114,7 +114,7 @@ echo "Installing Tracetest"
 echo "----------------------------"
 echo
 
-helm repo add kubeshop https://kubeshop.github.io/helm-charts
+helm repo add --force-update kubeshop https://kubeshop.github.io/helm-charts
 helm repo update
```

Comment:
Is this line still required? Or would the `force-update` flag already updates in any case?

Reply:
I'm not so sure, that's why I kept it... 
From the CLI docs "--force-update  -- replace (overwrite) the repo if it already exists"